PublicationsRouter/v2/oai-pmh/VERBS.md

```diff
@@ -138,7 +181,36 @@ Returned information:
 
 * Record: notification metadata, only oai_dc metadata prefix is supported
 
-PubRouter API request:
+Example request: `GET https://pubrouter.jisc.ac.uk/oaipmh/repo/123456789?verb=GetRecord&identifier=987654321&metadataPrefix=oai_dc`
```

Comment:
Have the URL on new line, so doesn't wrap (maybe shorten the identifier if needed).